build-system/pr-check/build-targets.js

```diff
@@ -26,13 +26,25 @@ const minimatch = require('minimatch');
 const path = require('path');
 const {gitDiffNameOnlyMaster} = require('../common/git');
 
+/**
+ * Checks if the given file is an OWNERS file
+ * @param {string} file
+ * @return {boolean}
+ */
+function isOWnersFile(file) {
+  return file.endsWith('OWNERS');
```

Comment:
Is `file` the full path or just the basename?

Reply:
It's the path relative to the repo root. For example: `build-system/tasks/OWNERS`.